app/models/address.rb

```diff
@@ -0,0 +1,4 @@
+class Address < ApplicationRecord
+  has_many :oders, dependent: :destroy
+  belongs_to :accounts
```

Comment:
phía model có quan hệ 1- thì em dùng số ít cho tên collection nhé
```suggestion
  belongs_to :account
```